scripts/flashing/gen_flashing_script.py

```diff
@@ -13,8 +13,11 @@
 # limitations under the License.!/usr/bin/env python
 """Generate script to flash or erase an EFR32 device."""
```

Comment:
is this still efr32-specific?  if so, can we keep efr32 in the name somewhere?  if not, can we remove the "EFR32" comments?